### ISSUE TITLE
Menus do not align with High Contrast 

### ISSUE THREAD

**Issue by ricklang** [Microsoft/vscode#6629]
- VSCode Version: 1.1.1
- OS Version: Windows Server 2012 R2

Steps to Reproduce:
1. Switch the color theme to High Contrast.
2. Click any menu item from the menu bar, the dropdown menu is displayed. But the background color is white, and the foreground (text) color is white too.
3. When mouse over the dropdown menu item, you can see the foreground color changes to black and you can see the menu itme.

**Comment by bpasero** [comment]
![image](https://cloud.githubusercontent.com/assets/900690/18613771/79160bb0-7d81-11e6-918b-451e458218c9.png)

@ricklang seems better now?

**Comment by ricklang** [comment]
Yes, it is better under standard Windows color theme.

If the Windows color theme is Hight Contrast Black, and the VSCode color
theme is High Contrast, still can't see the menu. But I can the shortcut
keys. VSCode Version: 1.5.2.

Also is it possible to adjust the background color (white) for the dropdown
menu based on color theme. There was a time (can't remember the version
number) that the background color is somewhat dark color.

Thanks.

Rick

[image: Inline image 1]

**Comment by bpasero** [comment]
Ok, it improved a lot (compared to before where you could not see menu entries), but still lacks some features from the windows HC theme.

**Comment by bpasero** [comment]
https://github.com/Microsoft/vscode/issues/17177
